src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double RADIUS = 10;
+    private static final Color COLOR = Color.WHITE;
+
+    enum RandomFigure {
+        CIRCLE,
+        ISOSCELES_TRAPEZOID,
+        RECTANGLE,
+        RIGHT_TRIANGLE,
+        SQUARE
+    }
+
+    private Random index = new Random();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+
+        switch (RandomFigure.values()[index.nextInt(RandomFigure.values().length)]) {
```

Comment:
extract expression from switch to a local variable for better readability
return switch expression itself